openapi/components/requestBodies/storefront/PostSubscriptionItemsChange.yaml

```
@@ -0,0 +1,92 @@
+type: object
+description: Change customer subscription items.
+properties:
+  effectiveTime:
+    type: string
+    description: Effective time when changes are applied.
+    default: now
+    enum:
+      - now
+      - next-service-period
+    x-enumDescriptions:
+      now: Apply changes immediately.
+      next-service-period: Apply changes at the beginning of next service period.
+  items:
+    type: array
+    description: Item details.
```

suggestion: for readability
```suggestion
    description: Details of items in the subscription.
```